subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -205,14 +214,18 @@ static bool validate_signature(u32_t fw_src_address,
 		return false;
 	}
 
-	PRINT("Firmware signature verified.\n\r");
+	if (validated) {
+		PRINT("Firmware signature verified.\n\r");
```

Comment:
Can this ever be executed? Since we have the `return false` on line 214 if `retval != 0` and we only set `validated` to `true` if `retval == 0` on line 199

Reply:
I suppose you mean whether the `else` clause can be executed. The answer is no. This is for the future. It could be an assert instead. Or I will add "should have had an error retval" to the print.

Reply:
Correct, also could we move this inside the scope of the `rot_verify` check? Where we already have checks on `validated`